CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Release 2.24.0 01-02-2021
+
+- Respond with HTTP 400 when required headers are omitted in CORS preflight request (#468)
+- Allow backend address override (#464)
+- Add an option to gcsrunner to set Envoy application log path (#467)
```

Comment:
Should be


- Respond with HTTP 400 when required headers are omitted in CORS preflight request (#468)
- Allow backend address override (#464)
- Propagate trace ID to correlate access logs and traces (#463)